PWGCF/FemtoUniverse/TableProducer/femtoUniverseProducerTask.cxx

```diff
@@ -1093,6 +1110,18 @@ struct femtoUniverseProducerTask {
   }
   PROCESS_SWITCH(femtoUniverseProducerTask, processTrackV0, "Provide experimental data for track v0", false);
 
+  void processTrackV0CentRun3(aod::FemtoFullCollisionCentRun3 const& col,
+                              aod::BCsWithTimestamps const&,
+                              soa::Filtered<aod::FemtoFullTracks> const& tracks,
+                              o2::aod::V0Datas const& fullV0s)
+  {
+    // get magnetic field for run
+    getMagneticFieldTesla(col.bc_as<aod::BCsWithTimestamps>());
+    // fill the tables
+    fillCollisionsAndTracksAndV0AndPhiCentRun3<false>(col, tracks, fullV0s);
```

Comment:
Maybe a better option for this process function would be creating particles that you really need in the analysis. Try doing something like:
-getMagneticField
-fillCollisions
-fillTracks
-fillV0